bittensor/__init__.py

```diff
@@ -42,12 +42,15 @@
 # Bittensor code and protocol version.
 __version__ = "7.0.0"
 
-version_split = __version__.split(".")
-__version_as_int__: int = (
-    (100 * int(version_split[0]))
-    + (10 * int(version_split[1]))
-    + (1 * int(version_split[2]))
+_version_split = __version__.split(".")
+__version_info__ = tuple(map(int, _version_split))
```

Comment:
let's use 
```python
__version_info__ = tuple(int(part) for part in _version_split)
``` 
instead of 
```python
__version_info__ = tuple(map(int, _version_split))
```

in this example it is the same in terms of optimization, but code readability is much better

Reply:
@RomanCh-OT I can make this change when this gets merged.

Reply:
Applied;

as for version being u32,  AFAIK, that was needed to be checked was in 
https://github.com/opentensor/subtensor/blob/0c77062a10cadfbab8507b7dccb2e1ea49848e58/pallets/subtensor/src/serving.rs#L57C9-L57C16
even with prometheus is u32 , but that version number is arbitrary set by node so it doesn't even matter in our context
https://github.com/opentensor/subtensor/blob/0c77062a10cadfbab8507b7dccb2e1ea49848e58/pallets/subtensor/src/serving.rs#L160

So it should be safe.